### PR TITLE
[MIRROR] Remplace sprite hammer with pickaxe

### PR DESCRIPTION
**Pull request by VirgoBot**
Original PR: https://github.com/PolarisSS13/Polaris/pull/5632
--------------------
This particular hammer had been sitting 'waiting for sprites' for 4 years at least. Could be a lot more really.

Fixes https://github.com/VOREStation/VOREStation/issues/4183
Virgo Port.
https://github.com/VOREStation/VOREStation/pull/4302

### CODE REVIEW

File: maps/southern_cross/southern_cross-3.dmm [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (SpadesNeil) [inline maps/southern_cross/southern_cross-3.dmm]: Bot can you _fucking not_?

**HOW IS THERE A CONFLICT?! WE DON'T EVEN TOUCH THIS FILE!!!**

**Comment by SpadesNeil** [comment]
Whatever, it looks messy in Github but only like a couple lines actually changed.

**Comment by Arokha** [comment]
But why edit their map at all

**Comment by SpadesNeil** [comment]
Because the bot is fucking dumb and makes PRs with conflicts.

**Comment by SpadesNeil** [comment]
Evidently, we still can't merge this because /hammer is on our map for some reason.

**Comment by lbnesquik** [comment]
I can't modify this pr anymore

**Comment by Arokha** [comment]
why would you have been able to

**Pull request merged**